src/components/fido-passkey-details-card/fido-passkey-details-card.stories.ts

```diff
@@ -10,29 +10,31 @@ export default {
         layout: 'centered',
     },
     argTypes: {
-        savedText: {
-            description: "Text to indicate the Created Date"
-        },
-        lastUsedText: {
-            description: "Text to indicate the Last Used Date"
-        },
-        lastUsedIsMobile: {
-            description: "Boolean to indicate if the Last Usage was on Mobile",
-            table: {defaultValue: {summary: 'false'}}
-        },
-        prevLastUsedText: {
-            description: "[Optional] Text to indicate the Previous Last Used Date"
-        },
-        prevLastUsedIsMobile: {
-            description: "[Optional] Boolean to indicate if the Previous Last Usage was on Mobile",
-            table: {defaultValue: {summary: 'false'}}
+        passkeyDetails: {
+            description: `PasskeyDetails Object:
+            
+                PasskeyDetails {
+                    createdAt: string,
+                    isSync: boolean,
```

Comment:
@kevin-villalobos-trusona is there a reason we didn't just use `synced` as the field name here? 

I'm not sure if we need to bring is java-idioms to typescript.